### PR TITLE
[OBSDEF-6350] Add default args for pre-upgrades

### PR DESCRIPTION
**Pull request by tmarcu**
The pre-upgrade checks need a `-target-version` argument to know which
version a user is attempting to upgrade to. In the health check screen,
there is no option to choose a version, only a health check to run.
Default to using the current version so that the check can still run
successfully when called without arguments.

Signed-off-by: Tudor Marcu <Tudor.Marcu@emc.com>

## Purpose
[OBSDEF-6350](https://jira.cec.lab.emc.com:8443/browse/OBSDEF-6350)

## PR checklist
- [ ] make test passed
- [ ] make build passed
- [ ] helm install <chart> passed
- [ ] vSphere7 make package deployments passed
- [ ] helm upgrade <chart> passed
- [ ] helm test <release_name> passed

## Testing
_Paste the output of your helm install/vSphere7 deployment testing here_

### CODE REVIEW

File: ecs-cluster/templates/objectstore-app-configmap.yaml [hunk 1]
```diff
@@ -38,3 +38,6 @@ data:
           container: {{.Values.global.registry}}/{{.Values.healthChecks.preUpdate.image.repository}}:{{default .Values.tag .Values.healthChecks.preUpdate.image.tag}}
           serviceaccount: {{.Release.Name}}-healthchecks
           timelimit: "5m"
+          args:
+            - -target-version
+            - {{ .Values.tag }}
```
Comment (vgupta-mickey) [inline ecs-cluster/templates/objectstore-app-configmap.yaml:43]: this shouldn't be {{default .Values.tag .Values.healthChecks.preUpdate.image.tag}

File: objectscale-manager/templates/objectscale-manager-app-configmap.yaml [hunk 2]
```diff
@@ -24,6 +24,9 @@ data:
           container: {{.Values.global.registry}}/objectscale-manager-pre-update:{{.Values.tag}}
           serviceaccount: {{ .Release.Name }}-healthchecks
           timelimit: "5m"
+          args:
+            - -target-version
+            - {{ .Values.tag }}
```
Comment (vgupta-mickey) [inline objectscale-manager/templates/objectscale-manager-app-configmap.yaml:29]: this shouldn't be {{default .Values.tag .Values.healthChecks.preUpdate.image.tag}

**Review by vgupta-mickey — APPROVED** [review]
It appears to me that pre-upgrade healthcheck must run against the targeted version, so ideally before we run the healthcheck, the healthcheck screen should pop-up the target version against which  the pre-upgrade health check is running. If the target version is not present, then it should fail with "insufficent information to run the pre-upgrade".

**Review by vgupta-mickey — COMMENTED** [review]
reverting the approve. It is done by mistake.

**Review by vgupta-mickey — DISMISSED** [review]
I have provided some comments, let me know if those make sense.

**Review by ben-schumacher — APPROVED** [review]
Discussed with Vinod.  Even though this isn't our ultimate solution, it sounds like it will allow us to close out the defect and not have it opened right back up again.

**Review by vgupta-mickey — APPROVED** [review]
Please open a story to run pre-upgrade only when the upgrade is required.

**Pull request merged**